nipy/modalities/fmri/experimental_paradigm.py

```diff
@@ -48,13 +48,7 @@ def __init__(self, con_id=None, onset=None, amplitude=None):
         self.amplitude = amplitude
         if con_id is not None:
             self.n_events = len(con_id)
-            try:
```

Comment:
I'm still worried by backwards compatibility - I think previous behavior was:

```
import nipy.modalities.fmri.experimental_paradigm as ep
In [2]: ep.Paradigm([1, 2, 3]).con_id
Out[2]: 
array(['c1', 'c2', 'c3'], 
      dtype='|S2')
```

where behavior on this branch is:

```
In [3]: ep.Paradigm([1, 2, 3]).con_id
Out[3]: 
array(['1', '2', '3'], 
      dtype='|S1')
```

I can see that it's a good idea not to convert strings to ints then back to strings, but maybe preserve `c` prefix if condition names are integers?